ci/tests/sonarqubetest/features/sonarqube_auth.feature

```diff
@@ -0,0 +1,28 @@
+Feature: sonarqube authentication and authorisation
+  In order to have a valid sonarqube authentication
+  As a sonarqube user
+  I am able to view the assigned project
+
+  @unit
+  Scenario: Can access login screen
+    Given there is a sonarqube install
+    When I access the login screen
+    Then sonarqube should be unlocked
+
+  @unit
+  Scenario: Can login as test standalone user
+    Given there is a sonarqube install
+    When I access the login screen
+    Then I am able to login
+
+  @unit 
+  Scenario: Unable to login without valid credentials
+    Given there is a sonarqube install
+    When I access the login screen
+    Then I am unable to login
+
+  @integration
+  Scenario: Unable to login as anonymous user
+    Given there is a sonarqube install
+    When I access the login screen
+    Then I am presented with a Github login option
```

Comment:
Fine to leave this for now, but when @cagiti and @macox are back it might be worth discussing where we host the integration tests.

Reply:
I presume we've got this covered because we are using tags @unit and @integration. What do you think?